src/Controller/PictureController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Picture;
+use App\Entity\Villa;
+use App\Form\PictureType;
+use App\Repository\PictureRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/picture")
+ */
+class PictureController extends AbstractController
+{
+    /**
+     * @Route("/", name="picture_index", methods={"GET"})
+     */
+    public function index(PictureRepository $pictureRepository): Response
+    {
+        return $this->render('picture/index.html.twig', [
+            'pictures' => $pictureRepository->findAll(),
+        ]);
+    }
+
+    /**
+     * @Route("/new", name="picture_new", methods={"GET","POST"})
+     */
+    public function new(Request $request): Response
+    {
+        $picture = new Picture();
+        $form = $this->createForm(PictureType::class, $picture);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $entityManager = $this->getDoctrine()->getManager();
```

Comment:
You should inject the entityManager using dependecy injection in the controller's parameters.